modules/core/src/main/java/org/apache/ignite/internal/processors/cluster/ChangeGlobalStateMessage.java

```diff
@@ -71,6 +71,9 @@
     @GridToStringExclude
     @Nullable private transient ServiceDeploymentActions serviceDeploymentActions;
 
+    /** Forced change of cluster state. */
+    private boolean force = true;
```

Comment:
Why the default value is true?

Reply:
> Why the default value is true?

For backward compatibility. Most of the deactivation requests are still forcable. Through Ignite, IgniteMXBean, IgniteCluster etc.

Reply:
I think we should make `force == false` by default and explicitly pass `true` when required.